scripts/bumpPackages/index.js

```diff
@@ -29,7 +29,7 @@ const runExec = async (version, packageDir) => {
 
   return new Promise((resolve, reject) => {
     exec(
-      `yarn version --${versionTag} --no-git-tag-version`,
+      `yarn version ${versionTag}`,
```

Comment:
this definitely still doesn't add any git tags?

Reply:
From what I've read in the documentation I can see no mention of git tags https://yarnpkg.com/features/release-workflow. But I'm not 100% sure, would it be best if I test with this command manually?

Reply:
what made you remove it? was it causing an error?

Reply:
The new plugin for `yarn version` doesn't support the --no-git-tag-version flag, and would error when trying to run it. And when running the package bumping script no git tags are added in the `.git/ref/tags` folder